config/webpack.config.js

```diff
@@ -33,7 +44,80 @@ const globalEntryFiles = {
   ],
 };
 
-const configGenerator = (buildOptions, apps) => {
+/**
+ * Get a list of all the entry points.
+ *
+ * If the `entry` CLI argument is passed, only the specified
+ * application entries are built.
+ */
+function getEntryPoints(entry) {
```

Comment:
I had these in the experimental `standalone-webpack.config.js` which I renamed to `webpack.config.js`. Essentially, this lays the groundwork for working with multiple `manifest.json` files.

Reply:
> lays the groundwork for working with multiple manifest.json files.

The build process already works with multiple manifest.json files. How his this different?

Reply:
It already works with multiple manifest.json files because of these "manifest helpers." I just copied them into this file because we'll be removing `src/site/` down the road and I didn't want any dependencies in there.

Reply:
Ah, I remember now. I copied this from [`build/webpack/index.js`](https://github.com/department-of-veterans-affairs/vets-website/blob/7ff9874b2da7eef36997a9033ee96c043c5a915f/src/site/stages/build/webpack/index.js#L18). :sweat_smile: 

I still didn't author it. Just misremembered where I copied it from.